codegen/templates/http_client.tmpl

```diff
@@ -98,22 +99,79 @@ func (c *{{$clientName}}) {{title .Name}}(
 		respHeaders[k] = res.Header.Get(k)
 	}
 
-	res.CheckOKResponse({{.OKStatusCode.Code}})
+	res.CheckOKResponse([]int{
```

Comment:
There is no reference to .ValidStatusCodes anywhere. it only exists on EndpointSpec which is not loaded in the gateway binary only in the code generation runner.